interface/VideoCommonDefs.h

```diff
@@ -33,12 +33,14 @@
 #endif
 namespace YamiMediaCodec{
 #ifdef ANDROID
+    #define BIND std::bind
     #define SharedPtr std::shared_ptr
     #define WeakPtr std::weak_ptr
     #define DynamicPointerCast std::dynamic_pointer_cast
     #define StaticPointerCast std::static_pointer_cast
     #define EnableSharedFromThis std::enable_shared_from_this
 #else
+    #define BIND std::tr1::bind
```

Comment:
Why all uppercase?  It's inconsistent with the rest of the naming.  How about "Bind" instead?